spec/services/section/new_service_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+RSpec.describe Section::NewService do
+  it 'returns the manual and a new section' do
+    user = double(:user)
+    section = double(:section)
+    manual = double(:manual, build_section: section)
```

Comment:
I've added a second commit to this branch ("Use real objects in the test") that uses real objects instead of stubs. Can you take a look and let me know whether you think it's any better please, @floehopper?